src/Multiselect.vue

```diff
@@ -111,11 +111,11 @@
         </slot>
       </template>
 
-      <slot v-if="!hasSelected && caret && !busy" name="caret">
+      <slot v-if="(!replaceCaretWithClear || replaceCaretWithClear && !hasSelected) && caret && !busy" name="caret">
```

Comment:
@mseele should this variable be named `hideCaretOnSelect` ?

Reply:
Good idea. The problem is that the variable is responsible for displaying the clear action, too. I don't know if this is clear with the other name.
But I'm also mit very happy with my Name, btw.

Reply:
Yeah, I'm not sure what's the best name but was just wondering that the `.d.ts` prop name is different from this one

Reply:
You're write, I will change the .d.ts name tomorrow

Reply:
type definition name is fixed